src/goto-analyzer/static_verifier.h

```diff
@@ -36,4 +38,34 @@ void static_verifier(
   const ai_baset &ai,
   propertiest &properties);
 
+enum class ai_verifier_statust
```

Comment:
:bulb: Precise definitions of these entries would be great.

Reply:
Documented.

In doing so I noticed a potential bug here:
https://github.com/diffblue/cbmc/blob/b5d2a7283b39b721f0e5f811d43a8568167879bf/src/goto-analyzer/static_verifier.cpp#L272

As far as I can tell this code is not used anywhere in-tree.